ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -305,7 +305,7 @@ Item {
             visible: {
                 return gapNowAndOldest < maxGapInSeconds
                         && gapNowAndJoined > maxGapInSeconds
-                        && (chatsModel.activeChannel.chatType !== Constants.chatTypePrivateGroupChat || chatsModel.activeChannel.isMember)
+                        && (chatsModel.channelView.activeChannel.chatType !== Constants.chatTypePrivateGroupChat || chatsModel.channelView.activeChannel.isMember)
```

Comment:
this can in fact fix the new bug about "load more messages" invisible...
thanks!!!